stripe/src/main/java/com/stripe/android/net/StripeSSLSocketFactory.java

```diff
@@ -0,0 +1,120 @@
+package com.stripe.android.net;
+
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocket;
+import javax.net.ssl.SSLSocketFactory;
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Wraps a SSLSocketFactory and enables more TLS versions on older versions of Android.
+ * Most of the code is taken from stripe-java.
+ */
+public class StripeSSLSocketFactory extends SSLSocketFactory {
+
+    private final SSLSocketFactory under;
```

Comment:
I went back and forth on whether or not I should convert this class to Hungarian notation as well. At the moment, I opted against it because I'm filing this class under "90% copied from stripe-java and I never intend to change anything about it ever" because I only superficially understand what's going on.